blog-style.css

```diff
@@ -0,0 +1,74 @@
+.blog {
+  background-color: #252525;
+  text-align: center;
+  color: white;
+  font-family: 'Poller One', cursive;
+  margin: 2%;
+  padding: 1% 3%;
+}
+.title {
+  display: flex;
+  justify-content: center;
+}
+.photo {
+  height: 300px;
+  width: 380px;
+  border-right: 2px solid #252525;
+}
+.blogitem {
+  display: flex;
+  border: 2px solid #252525;
+  margin: 1% 5%;
+}
+.text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 5%;
+  text-align:center;
+}
+.text h2 {
+  margin-bottom: 4%;
+}
+.text a {
+  text-decoration: none;
+  background-color: #252525;
+  color: white;
+  padding: 1% 4%;
+  margin-top: 4%;
```

Comment:
using percentage units for padding and margins will work and give you what you want on your specific browser size. but you may see big differences on a larger browser or smaller. bc it's a percent of the parent/child element. and if the child/parent element is also a percentage then you really have no control over how your component will look different sized browsers. 

i'm not saying not to use percentages but just be aware that spacing will change if the parent/child element size changes due to container size.